src/greyVest/TableFooter.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import Pager from './Pager'
+import PagerItem from './PagerItem'
+import PageSize from './PageSize'
+import Flex from './Flex'
+
+let showing = (totalRecords, page, pageSize) => {
+  let min = _.min([totalRecords, pageSize * (page - 1) + 1])
+  let max = _.min([totalRecords, pageSize * page])
+  return min >= max ? max : `${min}-${max}`
```

Comment:
This check is confusing - `min > max` feels like a bug even if it makes logical sense